alcs-api/src/common/authorization/authorization.service.ts

```diff
@@ -86,4 +108,34 @@ export class AuthorizationService {
 
     return res.data;
   }
+
+  // FIXME: this will be replaced with automapper
+  private mapUserFromTokenToCreateDto(
+    user: UserFromToken,
+  ): CreateOrUpdateUserDto {
+    return {
+      email: user.email,
+      name: user.name,
+      displayName: user.display_name,
+      identityProvider: user.identity_provider,
+      preferredUsername: user.preferred_username,
+      givenName: user.given_name,
+      familyName: user.family_name,
+      idirUserGuid: user.idir_user_guid,
+      idirUserName: user.idir_username,
+    } as CreateOrUpdateUserDto;
```

Comment:
You shouldn't need the as here if you are mapping correctly, this is kinda forcing it

Reply:
the difference is in CamelCase and SnakeCase. after casting object remained empty (only default fields populated)

Reply:
done